internal/api/group.go

```diff
@@ -21,54 +21,28 @@ func initGroup(apiRouter *mux.Router, context *Context) {
 	groupRouter.Handle("", addContext(handleGetGroup)).Methods("GET")
 	groupRouter.Handle("", addContext(handleUpdateGroup)).Methods("PUT")
 	groupRouter.Handle("", addContext(handleDeleteGroup)).Methods("DELETE")
-
-	groupRouter.Handle("/mattermost/{version:[A-Za-z0-9.]+}", addContext(handleChangeGroupMattermostVersion)).Methods("PUT")
 }
 
-// handleChangeGroupMattermostVersion responds to PUT
-// /api/group/{group}/mattermost/{version}, upgrading or downgrading
-// the installation to the Mattermost version embedded in the request.
-func handleChangeGroupMattermostVersion(c *Context, w http.ResponseWriter, r *http.Request) {
```

Comment:
I guess the right way to change this setting is now through using `handleUpdateGroup`?

Reply:
Yep, that's exactly right! The version endpoint was from the original group code where we didn't see a need for anything other than version so we streamlined it.

The cloud client hasn't actually used that endpoint in a while.

Reply:
Perfect :+1: